### PR TITLE
Painting Cleanup V2

### PR DESCRIPTION
**Pull request by AnturK**
This time without nuking categories. (Sorry basil)

### CODE REVIEW

File: code/controllers/subsystem/persistence.dm [hunk 1]
```diff
@@ -410,3 +410,31 @@ SUBSYSTEM_DEF(persistence)
...
 		ending_human.client.prefs.save_character()
+
+// Removes paintings with invalid md5
+/datum/controller/subsystem/persistence/proc/prune_persistent_paintings()
+	var/list/invalid = list()
+	for(var/pid in SSpersistence.paintings)
+		var/list/pruned_category = list()
+		var/list/current = SSpersistence.paintings[pid]
+		if(!current)
+			current = list()
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/persistence.dm:421]: why not just return here?

Comment (stylemistake) [reply code/controllers/subsystem/persistence.dm:421]: Because more stuff happens below

Comment (AnturK) [reply code/controllers/subsystem/persistence.dm:421]: It's just to ensure even empty category keys end up in final json.

Comment (LemonInTheDark) [reply code/controllers/subsystem/persistence.dm:421]: Oh I see. Thanks

**Comment by AnturK** [comment]
Pruning done so closing this

**Pull request closed without merging**